src/viur/core/utils.py

```diff
@@ -273,6 +273,29 @@ def normalizeKey(key: Union[None, 'db.KeyClass']) -> Union[None, 'db.KeyClass']:
     return db.Key(key.kind, key.id_or_name, parent=parent)
 
 
+def is_prefix(name: str, prefix: str, delimiter: str = ".") -> bool:
+    """
+    Utility function to check if a given name matches a prefix,
+    which defines a specialization, delimited by `delimiter`.
+
+    In ViUR, modules, bones, renders, etc. provide a kind or handler
+    to classify or subclassify the specific object. To securitly
+    check for a specific type, it is either required to ask for the
+    exact type or if its prefixed by a path delimited normally by
+    dots.
+
+    Example:
+
+    .. code-block:: python
+        handler = "tree.file.special"
+        utils.prefixes(handler, "tree")  # True
+        utils.prefixes(handler, "tree.node")  # False
+        utils.prefixes(handler, "tree.file")  # True
+        utils.prefixes(handler, "tree.file.special")  # True
```

Comment:
The examples must then be adapted:

```suggestion
        utils.is_prefix(handler, "tree")  # True
        utils.is_prefix(handler, "tree.node")  # False
        utils.is_prefix(handler, "tree.file")  # True
        utils.is_prefix(handler, "tree.file.special")  # True
```